src/controllers/shared/common.resume.submission.controller.ts

```diff
@@ -10,7 +10,13 @@ import { setExtraData } from "../../utils/application.data";
 import { isActiveFeature } from "../../utils/feature.flag";
 import { getOverseasEntity } from "../../service/overseas.entities.service";
 
-import { HasSoldLandKey, ID, IsSecureRegisterKey, OverseasEntityKey, Transactionkey } from "../../model/data.types.model";
+import {
+  HasSoldLandKey,
+  ID,
+  IsSecureRegisterKey,
+  OverseasEntityKey,
+  Transactionkey
+} from "../../model/data.types.model";
```

Comment:
I undid the changes to the resume controller, which were setting default values for is_listed_as_property_owner, I assumed we'd work on the resuming in a different ticket.

Reply:
Yes, a number of things to do to get resume working.